### DISCUSSION TITLE
好臭的台词啊啊啊啊啊啊！

### DISCUSSION THREAD

**Discussion by Doubudou45** [share121/inter-knot#5806] [公告]
不对劲不对劲
![Screenshot_20241221_014431](https://github.com/user-attachments/assets/5e1d0e6a-053d-44da-93d0-395270f5bc73)

**Comment by zgyhim** [comment]
114514
<img src="https://bbs-static.miyoushe.com/static/2024/10/28/254e0c5b298dccc4dda96a520a9fe808_9205129965460062091.png" alt="茜特菈莉-生气" width="75" >

**Comment by HardSkyml** [comment]
怎么在哪都能看到homo绳匠（恼）

**Comment by HJKexe** [comment]
无处不在的homo

**Comment by AsabaHarumasa567** [comment]
在现场，看着课长就冲过去了，根本追不上

**Comment by yuguang1351** [comment]
怎么哪里都有homo啊😱
